README.md

```diff
@@ -1,42 +1,61 @@
 # BullMQ Proxy
 
-This is a lightweight and fast proxy for BullMQ that allows you to access your queues via websockets. As BullMQ is currently a library available only for NodeJS and Python, this proxy allows you to access your queues from any language that supports websockets.
+This lightweight service works as a proxy for BullMQ queues. It has applications in many useful cases:
 
-Note: this is a work in progress and it is not ready for production yet.
+- Work with your queues in any language or platform that supports HTTP.
+- Run your workers in serverless environments.
+- Isolate your Redis instances and only allow BullMQ operations from untrusted sources.
```

Comment:
Yes. An untrusted source but it will still require a credential. But the "only" is wrong, it should be:
Isolate your Redis instances and allow BullMQ operations from untrusted sources.
For example, an untrusted source could be a webapp, adding a job directly from the browser.

Reply:
yeah, that would make this much clearer